6.6/scripts/oom_solr.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# This is just a wrapper to make it actually work in the start script - because
+# the whitespaces of belows call cause havoc otherwise:
+#. /opt/solr/bin/oom_solr.sh ${SOLR_PORT:-8983} /opt/solr/server/logs
+
+SOLR_LOGS_DIR=/opt/solr/server/logs
+SOLR_PID=`ps auxww | grep start.jar | grep -v grep | awk '{print $2}'`
```

Comment:
Use `$()` instead of backticks for greater readability and consistency with L14

Reply:
When testing I wanted to be as close as possible to the original handler: https://github.com/apache/lucene-solr/blob/master/solr/bin/oom_solr.sh
However, I've incorporated all your feedback into the latest update.